packages/ui5-proxy-middleware/README.md

```diff
@@ -5,15 +5,27 @@ The `@sap-ux/ui5-proxy-middleware` is a [Custom UI5 Server Middleware](https://s
 ## Configuration Options
 | Option       | Default Value | Description |
 | ------------ | ------------- | ----------- |
-| `ui5`        | mandatory     | List of mount paths and target urls that should be handled by the proxy |
+| `ui5`        | `object`      | List of mount paths and target urls that should be handled by the proxy. If not provided then `/resources` and `/test-resources` are proxied to `https://ui5.sap.com` |
```

Comment:
for `ui5-local.yaml` the proxy is configured without the ui5 option. Won't this default now be applied which wouldn't really be expected in offline mode.

Reply:
@devinea this default is only used when using the open source `ui5-proxy-middleware` directly in the yaml. The integrated version in `@sap/ux-ui5-tooling` is still in control of the used settings. And, yes, you are completely correct, for backward compatibility the `fiori-tools-proxy` can sadly not introduce this as default

Reply:
OK. Thanks for clarification.